ephios/extra/management/commands/setupdata.py

```diff
@@ -328,6 +345,135 @@ def create_objects(self, *args, **options):
+            title=_("Diving"),
+            uuid=uuid.UUID("6212a425-5cdf-49b0-bcb2-5075862a0c4a"),
+        )
+
+        signal = Qualification.objects.create(
+            category=diving_category,
+            title="Signalmann",
+            abbreviation="Signalmann",
+            uuid=uuid.UUID("436aaab0-5e8f-43aa-8f33-7d9ea68f7973"),
+        )
+
+        taucher_1 = Qualification.objects.create(
+            category=diving_category,
+            title="Einsatztaucher Stufe 1",
+            abbreviation="ET1",
+            uuid=uuid.UUID("52fc6a88-6e56-4af5-8021-b8a6412f405c"),
+        )
+        taucher_1.included_qualifications.add(signal)
+
+        taucher_2 = Qualification.objects.create(
+            category=diving_category,
+            title="Einsatztaucher Stufe 2",
+            abbreviation="ET2",
+            uuid=uuid.UUID("481fc9fb-e90a-4b83-8a07-456034aa0a33"),
+        )
+        taucher_2.included_qualifications.add(taucher_1)
+
+        tauchf = Qualification.objects.create(
+            category=diving_category,
+            title="Taucheinsatzführer",
+            abbreviation="TEF",
+            uuid=uuid.UUID("a3c5a3a0-f577-4bf5-8b2f-84108022d793"),
+        )
+        tauchf.included_qualifications.add(taucher_2)
+
+        boats_category = QualificationCategory.objects.create(
+            title=_("Boats (DLRG)"),
+            uuid=uuid.UUID("f1e2619b-56ee-4e4a-b429-b923f8586771"),
+        )
+        Qualification.objects.create(
+            category=boats_category,
+            title="DLRG-Bootsführerschein A",
+            abbreviation="BF Binnen",
+            uuid=uuid.UUID("7ec0c9e2-f211-4370-b796-e309f3e9b448"),
+        )
+        Qualification.objects.create(
+            category=boats_category,
+            title="DLRG-Bootsführerschein B",
+            abbreviation="BF See",
+            uuid=uuid.UUID("1ba25ee1-fa88-4d44-9416-3a024caa6daf"),
+        )
+
+        wr_category = QualificationCategory.objects.create(
+            title=_("Water rescue (DLRG)"),
+            uuid=uuid.UUID("93574a61-1a6e-4dd5-8df3-40e8c51b693f"),
+        )
+        rettschw_bronze = Qualification.objects.create(
+            category=wr_category,
+            title="Rettungsschwimmer Bronze",
+            abbreviation="RS Bronze",
+            uuid=uuid.UUID("247fab6a-8784-4976-a406-985fe47dc683"),
+        )
+        rettschw_silber = Qualification.objects.create(
+            category=wr_category,
+            title="Rettungsschwimmer Silber",
+            abbreviation="RS Silber",
+            uuid=uuid.UUID("ef95a854-2eeb-431c-a795-bc291b341d49"),
+        )
+        rettschw_silber.included_qualifications.add(rettschw_bronze)
+        rettschw_gold = Qualification.objects.create(
+            category=wr_category,
+            title="Rettungsschwimmer Gold",
+            abbreviation="RS Gold",
+            uuid=uuid.UUID("b601a18b-cee8-4037-af33-dd7aabeac295"),
+        )
```

Comment:
Rettungsschwimmen is not specific to DLRG. we could do a seperate DRSA-Catergory